packages/camera/camera_android_camerax/README.md

```diff
@@ -1,3 +1,93 @@
-# camera_android_camerax
+# camera\_android\_camerax
 
-An implementation of the camera plugin on Android using CameraX.
+An Android implementation of [`camera`][1] that uses the [CameraX library][2].
+
+*Note*: This package is under development.
+See [missing features and limitations](#missing-features-and-limitations).
+
+## Usage
+
+This package is [non-endorsed][3]; the endorsed Android implementation of `camera`
+is [`camera_android`][4]. To use this implementation of the plugin instead of
+`camera_android`, you will need to specify it in your `pubsepc.yaml` file as a
+dependency in addition to `camera`:
+
+```yaml
+dependencies:
+  # ...along with your other dependencies
+  camera: ^0.10.4
+  camera_android_camerax: ^0.5.0
+```
+
+## How this plugin accesses Android libraries
+
+The `camera` implementation is located at `./lib/src/android_camera_camerax.dart`, and
+it is implemented using Dart classes that are wrapped versions of Android classes.
+
+In `lib/src/`, you will find all of the Dart-wrapped native classes that the plugin
+currently uses to implement `camera`. Each of these classes uses an `InstanceManager`
+(implementation in `instance_manager.dart`) to manage objects that are created by
+the plugin implementation that map to objects of the same type created on the native
+side. This plugin uses [`pigeon`][12] to communicate with native code, so each of
```

Comment:
Here is where you could include the location of the pigeon file as well as the exact command line to run. See: https://pub.dev/packages/webview_flutter_android#contributing

Reply:
Ideally we should actually move that to a CONTRIBUTING.md; it's not relevant to plugin clients.